src/appengine/handlers/cron/triage.py

```diff
@@ -250,11 +257,94 @@ def _check_and_update_similar_bug(testcase, issue_tracker):
   return False
 
 
-def _file_issue(testcase, issue_tracker):
+def _get_job_bugs_filing_max(job_type):
+  """"Gets the maximum number of bugs that can be filed for a given job."""
+  if job_type in _bug_filing_max_24_hours_per_job:
+    return _bug_filing_max_24_hours_per_job[job_type]
+
+  max_bugs = None
+  job = data_types.Job.query(data_types.Job.name == job_type).get()
+  if job and 'BUG_FILING_MAX_24_HOURS_PER_JOB' in job.get_environment():
+    try:
+      max_bugs = int(job.get_environment()['BUG_FILING_MAX_24_HOURS_PER_JOB'])
+    except Exception:
+      logs.log_error(
+          'Invalid environment value of \'BUG_FILING_MAX_24_HOURS_PER_JOB\' '
+          f'for job type {job_type}.')
+
+  _bug_filing_max_24_hours_per_job[job_type] = max_bugs
+  return max_bugs
+
+
+def _get_project_bugs_filing_max():
```

Comment:
I think you can accomplish the same thing without using globals, by using functools.lru_cache(1) instead.